Dockerfile

```diff
@@ -101,6 +103,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-instal
     texlive-latex-recommended \
     texlive-science
 
+# spead2_net_raw is needed so that qualification tests can use ibverbs
+RUN wget https://raw.githubusercontent.com/ska-sa/spead2/master/src/spead2_net_raw.cpp && \
```

Comment:
Using master is not ideal - it means we're not isolated from changes on spead2. https://github.com/ska-sa/katgpucbf/commit/d842d297a0fd509d4b8fcbc42914cd82e891ec74 shows how an older version of the unit test Jenkinsfile used to extract the pinned version of spead2.

Reply:
Thanks for the tip.